packages/core/mesh/messaging/src/signal-manager/edge-signal-manager.ts

```diff
@@ -14,48 +16,61 @@ import {
   SwarmResponseSchema,
   type Message as EdgeMessage,
 } from '@dxos/protocols/buf/dxos/edge/messenger_pb';
-import { type SwarmEvent } from '@dxos/protocols/proto/dxos/mesh/signal';
 import { ComplexMap, ComplexSet } from '@dxos/util';
 
-import { type Message, type SignalMethods } from '../signal-methods';
+import { type SignalManager } from './signal-manager';
+import { type PeerInfo, type Message, type SwarmEvent, PeerInfoHash } from '../signal-methods';
 
 const SWARM_SERVICE_ID = 'swarm';
-export class EdgeSignal extends Resource implements SignalMethods {
-  public swarmEvent = new Event<{ topic: PublicKey; swarmEvent: SwarmEvent }>();
+const SIGNAL_SERVICE_ID = 'signal';
```

Comment:
There shouldn't be a singal service.
`serviceId` is used when you want to route a message to a service inside EDGE (e.g. replicator or swarm).
For messages between peers we should leave this field empty.